TestCases/compliance-level-3/0081-feel-getentries-function/0081-feel-getentries-function.dmn

```diff
@@ -0,0 +1,96 @@
+    <description>FEEL built-in function 'get entries(m)' in unspecified category</description>
+
+    <decision name="decision_001" id="_decision_001">
+        <description>Tests FEEL expression: 'get entries()' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries()'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_001"/>
+        <literalExpression>
+            <text>get entries()</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_002" id="_decision_002">
+        <description>Tests FEEL expression: 'get entries({a: "foo"}, {b: "bar"})' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries({a: "foo"}, {b: "bar"})'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_002"/>
+        <literalExpression>
+            <text>get entries({a: "foo"}, {b: "bar"})</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_003" id="_decision_003">
+        <description>Tests FEEL expression: 'get entries(null)' and expects result: 'null'</description>
+        <question>Result of FEEL expression 'get entries(null)'?</question>
+        <allowedAnswers>null</allowedAnswers>
+        <variable typeRef="list" name="decision_003"/>
+        <literalExpression>
+            <text>get entries(null)</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_004" id="_decision_004">
+        <description>Tests FEEL expression: 'get entries({a: "foo", b: "bar"})' and expects result: '[{a: "foo}, {b: "bar"}]'</description>
+        <question>Result of FEEL expression 'get entries({a: "foo", b: "bar"})'?</question>
+        <allowedAnswers>[{a: "foo}, {b: "bar"}]</allowedAnswers>
+        <variable typeRef="list" name="decision_004"/>
+        <literalExpression>
+            <text>get entries({a: "foo", b: "bar"})</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_005" id="_decision_005">
+        <description>Tests FEEL expression: 'get entries(m:{a: "foo", b: "bar"})' and expects result: '[{a: "foo}, {b: "bar"}]'</description>
+        <question>Result of FEEL expression 'get entries(m:{a: "foo", b: "bar"})'?</question>
+        <allowedAnswers>[{a: "foo}, {b: "bar"}]</allowedAnswers>
+        <variable typeRef="list" name="decision_005"/>
+        <literalExpression>
+            <text>get entries(m:{a: "foo", b: "bar"})</text>
```

Comment:
IIRC there are 2 tickets to the RTF one by yourself and one by myself, to ask to add these two functions in the Tabular section where parameters are specified. I am a bit doubtful if we really want to test by named-parameter-invocation before those RTF issues get fixed in the spec, despite from the textual description I concur parameters name are `m`, `key`

Reply:
Thanks @tarilabs.  I raised a few 'issue' tickets for the spec and then they didn;' seem to appear in the list so I thought I must have got it wrong somehow.  Glad they made it.  :-) 

What you say seems reasonable, I'll just comment out the named param assertions until things are concrete there.  Same for

Reply:
.. and same goes also for assertions `decision_007` and `decision_008` in `0080-feel-getvalue-function`

Reply:
@tarilabs. test updated as such.